include/flatbuffers/flatbuffers.h

```diff
@@ -1664,12 +1664,23 @@ const T *GetTemporaryPointer(FlatBufferBuilder &fbb, Offset<T> offset) {
 /// This function is UNDEFINED for FlatBuffers whose schema does not include
 /// a file_identifier (likely points at padding or the start of a the root
 /// vtable).
-inline const char *GetBufferIdentifier(const void *buf) {
-  return reinterpret_cast<const char *>(buf) + sizeof(uoffset_t);
+inline const char *GetSizePrefixedBufferIdentifier(const void *buf) {
+  return reinterpret_cast<const char *>(buf) + 2*sizeof(uoffset_t);
```

Comment:
use spacing `2 * ` like the rest of the code. For clarity, maybe better to call the other function than to repeat code.

Reply:
My interpretation was that basically, there shouldn't be too many additional functions for size prefixed buffers. Instead, the size prefix should be removed from the size prefixed buffer, and the resulting buffer should be used with the existing functions. This is why I had included the `GetSizePrefixedRoot` function.

Reply:
I've simplified the code as you mentioned.
Concerning code beauty, I've tried to run src/clang-format.sh, but it makes too many modifications (I don't know it is used anymore).

Reply:
We may not have clang-formatted in a while, so yeah, maybe leave that out for now. There is a way to use git to clang-format only your changes, which would be good